nats.go

```diff
@@ -4016,14 +4018,32 @@ func wipeSlice(buf []byte) {
 }
 
 func userFromFile(userFile string) (string, error) {
-	contents, err := ioutil.ReadFile(userFile)
+	path, err := expandTildePath(userFile)
+	if err != nil {
+		return _EMPTY_, fmt.Errorf("nats: %v", err)
+	}
+
+	contents, err := ioutil.ReadFile(path)
 	if err != nil {
 		return _EMPTY_, fmt.Errorf("nats: %v", err)
 	}
 	defer wipeSlice(contents)
 	return jwt.ParseDecoratedJWT(contents)
 }
 
+func expandTildePath(p string) (string, error) {
+	if !strings.HasPrefix(p, "~") {
+		return p, nil
+	}
+
+	home, err := os.UserHomeDir()
+	if err != nil {
```

Comment:
Should this just have ~/ -> /?

Reply:
Hm? Sorry, don't follow.

Reply:
if the homedir return an err should we just remove the ~?